graphql_compiler/tests/test_schema_generation.py

```diff
@@ -146,75 +146,135 @@
     ],
 })
 
+ABSTRACT_NON_GRAPH_CLASS_WITH_NON_VERTEX_CONCRETE_SUBCLASS = frozendict({
+    'name': 'AbstractNonGraphClassWithNonVertexConcreteSubclass',
+    'abstract': True,
+    'superClasses': [],
+    'properties': [],
```

Comment:
If `properties` is empty, then these are not representable in GraphQL regardless of everything else. Try to make the test data as "real" as possible, otherwise we can very easily end up with tests that pass by accident, rather than passing because the code is actually correct.